### DISCUSSION TITLE
Help getting the Galaxy Brain badge

### DISCUSSION THREAD

**Discussion by BEPb** [community/community#20730] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello everyone, I offer my help in obtaining this badge. I propose to add me to the followers and in my repositories ask 2 questions (create 2 discussions) to which I answer and you get this badge. After that, I create similar 2 discussions in your repositories and you answer them, so I will get the same badge. How do you like my idea? You can contact me by mail, the details are indicated on my page.

![](https://user-images.githubusercontent.com/51322709/178837162-fdb7880a-5435-4723-acc1-4abbbebf0de2.png)

**Comment by radar-blip** [comment]
And what kind of achievement would you call that? (just curious).

**Reply by BEPb** [reply]
This thread was not created to satisfy the curiosity of those who do not understand the motives for obtaining this badge, and I do not want to convince anyone of the need to do so. I am not calling for this. I ask for help and I am ready to provide it in return.

**Reply by radar-blip** [reply]
ok

**Reply by SparkScratch-P** [reply]
> And what kind of achievement would you call that? (just curious).

Involvement achievement!

**Comment by BEPb** [comment]
It remains for me to receive only this badge from the possible ones. I am also ready to share my experience and assist in obtaining other badges.

**Comment by thuongtruong1009** [comment, marked as answer]
Hi @BEPb 
This is a great idea. Can you cooperate with me! 
This is my [email](mailto:ititiu19228@student.hcmiu.edu.vn)

**Comment by PhyoYazar** [comment]
Hi @BEPb 
I like to get the badge and this is great idea.
Let's make it
Here my [email](phyoyazar592@gmail.com)

**Comment by github-staff** [comment]
We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
